frontend/pages/admin/IntegrationsPage/cards/CertificateAuthorities/components/EditCertAuthorityModal/EditCertAuthorityModal.tsx

```diff
@@ -92,6 +93,16 @@ const EditCertAuthorityModal = ({
     if (certAuthority.type === "hydrant") {
       return HydrantForm;
     }
+    if (certAuthority.type === "smallstep") {
+      return SmallstepForm;
+    }
+
+    // FIXME: seems like we have some competing patterns in here where we sometimes do switch
```

Comment:
yah think it makes sense to change to switch statements. The default is just there to not have to handle an undefined case. but we can also change it to how your recommending.